docs/v3.x/concepts/models.md

```diff
@@ -164,12 +164,18 @@ The options key on the model-json states.
 
 - `timestamps`: This tells the model which attributes to use for timestamps. Accepts either `boolean` or `Array` of strings where first element is create date and second element is update date. Default value when set to `true` for Bookshelf is `["created_at", "updated_at"]` and for MongoDB is `["createdAt", "updatedAt"]`.
 
+- `privateAttributes`: This configuration allows to treat as private a set of attributes, even if they're not actually defined as attributes in the model. Accepts an `Array` of strings. It could be used to remove from API responses timestamps or `_v` when using MongoDB. The set of `privateAttributes` defined in the model are merged with the `privateAttributes` defined in the global Strapi configuration only when `ignoreGlobalPrivateAttributes` is set to `false`.
+
+- `ignoreGlobalPrivateAttributes`: This configuration allows to ignore the global `privateAttributes`. The set of `privateAttributes` defined in the model won't be merged with the `privateAttributes` defined in the global Strapi configuration.
```

Comment:
TBH, I'm not sure about this param since it adds complexity in the way we manage private attributes (eg: sanitize-entity).

In my opinion, a user who wants to ignore global private params for at least one content type shouldn't add them to the global params list.

However, I can see the overhead it would be to be forced to re-declare some private fields for almost every content-type.

What about something like an `ignorePrivateFor` attribute in the model's options (eg: `ignorePrivateFor: ['foo', 'bar']`), which would force the attributes in the array to be considerate as non-private (whether it had been declared globally or locally in the model)?

(I'm open to suggestions for the `ignorePrivateFor` name)

Reply:
@petersg83, do you have an opinion about this?